airbyte-db/lib/configs_db_schema.md

```diff
@@ -0,0 +1,359 @@
+
+This table has many-to-one relationship with the `SYNC` table. Each `SYNC` can have multiple `SYNC_OPERATION`s.
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| workspace_id | char(36) | yes | | Foreign key. Index. |
+| name | varchar(255) | yes | | |
+| operator_type | varchar(50) | yes | | String enum: "normalization", "dbt". |
+| tombstone | boolean | | false | index | |
+
+### `NORMALIZATION_OPERATOR`
+
+YAML reference:
+- [OperatorNormalization](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/OperatorNormalization.yaml)
+
+This table has a one-to-one relationship with the `SYNC_OPERATION` table. Each `SYNC_OPERATION` has at most one `NORMALIZATION_OPERATOR`.
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| normalization_type | string | varchar(50) | "basic" | String enum: "basic". |
+
+### `DBT_OPERATOR`
+
+YAML reference:
+- [OperatorDbt](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/OperatorDbt.yaml)
+
+This table has a one-to-one relationship with the `SYNC_OPERATION` table. Each `SYNC_OPERATION` has at most one `DBT_OPERATOR`.
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| git_repo_url | varchar(1024) | yes | | |
+| git_repo_branch | varchar(255) | | | |
+| docker_image | varchar(255) | | | |
+| dbt_arguments | varchar(2048) | | | |
+
+### `SOURCE_OAUTH_PARAM`
+
+YAML reference:
+- [DestinationOAuthParameter](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/DestinationOAuthParameter.yaml)
+
+Many-to-one relationship with `SOURCE_DEFINITION`. Each `SOURCE_DEFINITION` can have multiple `SOURCE_OAUTH_PARAM`: global or workspace-specific.
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| source_definition_id | char(36) | yes | | Foreign key. Composite index. |
+| workspace_id | char(36) | | | Foreign key. Composite index. |
+| configuration | jsonb | yes | | |
+
+Composite index: (source_definition_id, workspace_id).
+
+### `DESTINATION_OAUTH_PARAMETER`
+
+YAML reference:
+- [DestinationOAuthParameter](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/DestinationOAuthParameter.yaml)
+
+Many-to-one relationship with `DESTINATION_DEFINITION`. Each `DESTINATION_DEFINITION` can have multiple `DESTINATION_OAUTH_PARAM`: global or workspace-specific.
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| destination_definition_id | char(36) | yes | | Foreign key. Composite index. |
+| workspace_id | char(36) | | | Foreign key. Composite index. |
+| configuration | jsonb | yes | | |
+
+Composite index: (source_definition_id, workspace_id).
+
+### `SYNC_INPUT`
```

Comment:
I'm not sure we need this table. I don't think this is information that we ever persist in the DB currently. Is there a reason we should add it?

Reply:
I think the same is true for `SYNC_INPUT_OPERATION_SEQUENCE`, `NORMALIZATION_INPUT`, `DBT_OPERATOR_INPUT`, and `SYNC_OUTPUT`